app/Handlers/GithubWebhookHandler.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Handlers;
+
+use App\Services\OrganizationService;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Validation\UnauthorizedException;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+class GithubWebhookHandler
+{
+    public function handle(Request $request): void
+    {
+        Log::info("Received webhook request:", $request->all());
+
+        $this->validateGithubWebhook(config("services.github.webhook_secret"), $request);
+
+        Log::info($request->getContent());
```

Comment:
Do u wanna log all data?

Reply:
i'm not sure, deleted logging for now. Data about webhook requests can be seen in github app panel.

Reply:
Maybe another question. For what we need to log that data?

Reply:
For monitoring and troubleshooting, I think.

Reply:
IMO, that's useless in that form, for handling exceptions or things like that u can use [Sentry](https://sentry.io/) but it will be good idea for future.